src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -126,6 +129,14 @@ class EmojiPickerMenu extends Component {
         this.setState({selection: event.nativeEvent.selection});
     }
 
+    /**
+     * Find and store index of the first emoji item
+     * @param {Array} filteredEmojis
+     */
+    setFirstNonHeaderIndex(filteredEmojis) {
+        this.firstNonHeaderIndex = _.findIndex(filteredEmojis, item => !item.spacer && !item.header);
```

Comment:
what happens when firstNonHeaderIndex is `-1` ?

Reply:
> what happens when firstNonHeaderIndex is `-1` ?

It could happen if there are no emojis in the list, at that point search input stays focused and keyboard navigation does nothing. However, if you try navigating with your keyboard while the emoji list is empty, error appears in the console. It is also present in production, so it wasn't introduced by this change.
I've added an early return at the top of `highlightAdjacentEmoji`, so it is not trying to highlight an emoji if there are none